code/src/containers/forms/new-product-form/NewProductForm.scss

```diff
@@ -0,0 +1,138 @@
+@import "@design-system";
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - 10px) calc(70% - 10px);
+  grid-gap: spacing(5);
+  box-sizing: border-box !important;
+  margin-bottom: spacing(5);
+
+  & div {
+    box-sizing: border-box !important;
+  }
+
+  &__header {
+    background-color: var(--products-table-head-bg-color, $light-alpha);
+    padding: spacing(5) spacing(8);
+  }
+
+  &__body {
+    padding: spacing(8);
+  }
+
+  &__container {
+    box-shadow: var(--products-table-shadow, $shadow-black);
+    background-color: white;
+  }
+
+  &__image-preview {
+    border: 1px solid var(--spa-accordion-container-border-color, $gray-light);
+    border-radius: 3pxs;
```

Comment:
Is it a typo?

Reply:
Thanks for noticing, that was a type. Fixed